tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "target": "ES6",
+    "target": "ES2018",
```

Comment:
Why do we need to update ES6 target ?

Reply:
This is probably for the private class field support. Anyway is ES2018 is OK for our browser targets?